test/cli/cli.go

```diff
@@ -127,6 +127,69 @@ func getAddCommand() *cobra.Command {
 		Short: "Add resources to the cluster",
 	}
 	cmd.AddCommand(getAddSimulatorCommand())
+	cmd.AddCommand(getAddNetworkCommand())
+	return cmd
+}
+
+// getAddNetworkCommand returns a cobra command for deploying a stratum network
+func getAddNetworkCommand() *cobra.Command {
```

Comment:
I know the current image deploys a 3 node instance of Stratum and I'm ok with that. Could you work with Carmelo and see if we can have a docker image of a single stratum device with no mininet added ?

Reply:
@Andrea-Campanella 
The idea is to allow the user to configure the number of nodes in the network when it sends a request to create a network pod. I mean it can be a single node or multiple nodes. But if you are looking for a completely standalone version of stratum to be executed outside mininet, I will check with Carmelo to see can we have one.

Reply:
@adibrastegarnia as per our discussion yesterday the network solution is fine. Parametrize the mininet launch script with the number of stratum nodes passed from the user.